src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -2,0 +1,23 @@
+import cn from 'classnames';
+import { UserInfo } from '../UserInfo';
```

Comment:
Verify that the `UserInfo` component is correctly defined and imported from the `../UserInfo` directory. It should be capable of rendering user details.